terraform/wallarm-rules/main.tf

```diff
@@ -0,0 +1,252 @@
+# 
+# Define provider parameters
+# 
+provider "wallarm" {
```

Comment:
How to define Wallarm customer account ID?

Reply:
client_id = var.wallarm_client_id

Reply:
`client_id = var.wallarm_client_id`
It is possible to define this parameter per resource.